scripts/run

```diff
@@ -17,6 +17,7 @@ INITRD_TMP=${BUILD}/$(sha1sum ${INITRD} | awk '{print $1}')
 INITRD_CURRENT=${BUILD}/initrd-current
 INITRD_TEST=${BUILD}/initrd.test
 USER_DATA=cloud-init/openstack/latest/user_data
+CLOUD_CONFIG=$(readlink -f "$1") || :
```

Comment:
There's already an arg parsing while loop lower in the script.  This will conflict with that.